pkg/version/version_test.go

```diff
@@ -209,6 +209,33 @@ func TestNext(t *testing.T) {
 			want:    "",
 			wantErr: true,
 		},
+		{
+			name: "bump major not knowing minor and patch",
+			args: args{
+				current:          "1.5.2",
+				incrementPattern: "x.?.?",
+			},
+			want:    "2.5.2",
```

Comment:
```suggestion
			want:    "2.0.0",
```
Usually, the minor and patch levels are not carried over to the next major.